x-pack/legacy/plugins/actions/README.md

```diff
@@ -23,6 +23,27 @@ action types.
 2. Create an action by using the RESTful API (see actions -> create action).
 3. Use alerts to execute actions or execute manually (see firing actions).
 
+## Kibana Actions Configuration
+Implemented under the [Actions Config](./server/actions_config.ts).
+
+### Configuration Options
+
+Built-In-Actions are configured using the _xpack.actions_ namespoace under _kibana.yml_, and have the following configuration options:
+
+| Namespaced Key                       | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Type                  |
+| ------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------- |
+| _xpack.actions._**enabled**          | ?                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | boolean               |
```

Comment:
For the `?`, perhaps we can find suitable text from some other plugin?  Basically, this is used to enable actions - I believe it will be `true` by default once we ship, but happens to be `false` right now due to some leakage in feature controls (actions and alerting show up in some ui spot in Kibana right now, if enabled).

Reply:
Yeah, I wasn't sure, left it that way to double check :)

Reply:
Updated docs